AdobeIms/Model/GetImage.php

```diff
@@ -18,6 +18,8 @@
  */
 class GetImage implements GetImageInterface
 {
+    private const IMAGE_SIZES = [50, 100, 115, 230, 138, 276];
```

Comment:
I'd remove any image present in the `$result['user']['images']` if the requested size is not available. In case the sizes returned are different from the sizes listed here